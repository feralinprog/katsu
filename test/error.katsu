# cause a signal to be raised
1 / 0
