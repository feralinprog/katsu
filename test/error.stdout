Error: could not load module test.
divide-by-zero: cannot divide by integer 0
at <src/core/core.katsu:316:1-324.2>
at <src/core/core.katsu:145:20-145.61>
at <src/core/core.katsu:41:23-41.52>
at <src/core/core.katsu:145:49-145.58>
at <src/core/core.katsu:317:5-317.98>
at <src/core/core.katsu:240:5-276.6>
at <src/core/core.katsu:243:31-263.10>
at <src/core/core.katsu:188:31-188.61>
at <src/core/core.katsu:180:5-186.6>
at <src/core/core.katsu:183:23-183.29>
at <src/core/core.katsu:255:13-262.14>
at <src/core/core.katsu:168:20-168.61>
at <src/core/core.katsu:41:23-41.52>
at <src/core/core.katsu:168:49-168.58>
at <src/core/core.katsu:258:17-258.78>
at <src/core/core.katsu:209:5-216.48>
at <src/core/core.katsu:168:20-168.61>
at <src/core/core.katsu:41:23-41.52>
at <src/core/core.katsu:168:49-168.58>
at <src/core/core.katsu:212:9-214.10>
at <src/core/core.katsu:14:9-14.18>
at <src/core/core.katsu:213:21-213.30>
at <test/error.katsu:2:1-2.6>
at <src/core/condition-handler.katsu:3:5-3.74>
